src/ecr_check_repos_for_terraform/ecr_check_repos_for_terraform/config.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+import argparse
+import logging
+
+import ecr_check_repos_for_terraform.context
+from lib.logging import setup_logging
+
+class AppConfig:
+  def __init__(self, cmdline_args, env):
+    ''' set app-wide configs and initialize the app '''
+    loglevel = (
+      cmdline_args.loglevel
+    )
+
+    self._init_app(loglevel)
+
+  def _init_app(self, loglevel):
+    ''' initialize the app '''
+    # initialize logging
+    setup_logging(eval('logging.' + loglevel))
+
+def parse_args():
+  ''' parse command line args '''
+  parser = argparse.ArgumentParser(
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter
+  )
+  parser.add_argument(
+    '--loglevel',
+    choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
+    default='INFO',
+    help='log level'
+  )
+  return parser.parse_args()
+
+def parse_env(env):
```

Comment:
Looks like this is unneeded

Reply:
correct. left it for easier adding env vars in the future, should we have to. but sure, let's [remove it](https://github.com/artsy/opstools/commit/24a58fbf814fe4e7ecfd7b669f9a0eca61f6a1c7), feels cleaner.